package.json

```diff
@@ -24,7 +24,7 @@
     "bracketSpacing": true,
     "semi": false,
     "singleQuote": true,
-    "trailingComma": "none"
+    "trailingComma": "es5"
```

Comment:
So odd -- when I click the "File Changes" tab in the PR this commit doesn't show up -- GH caching?